translations/en.json

```
@@ -133,6 +133,7 @@
   "msg.sync.started": "Course scan started.",
   "msg.sync.failed": "Course scan failed. Course is missing.",
   "msg.file.replace.file_type": "File type not accepted. Please input a file with the correct filetype.",
+  "msg.file.replace.file_size": "File size too large. Please input a file with a size less than 10MB",
```

Nit-picky, though I feel this wording might sound better? 

```suggestion
  "msg.file.replace.file_size": "File size too large. Please input a file of a size less than 10MB",
```